plugin-server/src/main/ingestion-queues/session-recording/utils.ts

```diff
@@ -246,8 +263,21 @@ export const parseKafkaMessage = async (
     let messagePayload: RawEventMessage
     let event: PipelineEvent
 
+    const compression = readCompressionFromHeader(message.headers)
+
+    let message_unzipped = message.value
```

Comment:
```suggestion
    let messageUnzipped = message.value
```

Reply:
ha, switching backwards and forward from python to TS kills me